docs/satellite-clone.md

```diff
@@ -91,3 +91,24 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Cloning a Satellite With Remote Databases
+
+Note: the backup used for this procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled configuration backups.
+
+1. Set up the remote databases according to Satellite documentation in the same way that they were set up on your original Satellite.
```

Comment:
perhaps hyperlink to https://access.redhat.com/documentation/en-us/red_hat_satellite/ ?